test/unit/behaviours/get-living-status.test.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const reqres = require('reqres');
+const Behaviour = require('../../../apps/right-to-rent-check/behaviours/get-living-status');
+
+describe('apps/right-to-rent-check/behaviours/get-living-status', () => {
+    it('exports a function', () => {
+      Behaviour.should.be.a('function');
+    });
+
+    class Base {
+      locals() {}
+    }
+    let req;
+    let res;
+    let sessionModel;
+    let instance;
+    let GetLivingStatus;
+    let translate;
+
+    const superLocals = {
+      tenants: [{'tenant-name': 'Mo Khan'}],
+    };
+
+    beforeEach(() => {
+      sessionModel = {
+        get: sinon.stub()
+      };
+      translate = sinon.stub();
+      res = reqres.res();
+      req = reqres.req({sessionModel, translate});
+      sinon.stub(Base.prototype, 'locals').returns(superLocals);
+      GetLivingStatus = Behaviour(Base);
+      instance = new GetLivingStatus();
+    });
+
+    afterEach(() => {
+      Base.prototype.locals.restore();
+    });
+
+    describe('locals', () => {
+      it('returns an extended locals with living status when a tenant is currently living in the property', () => {
```

Comment:
Any thought of a better name for this test?

Reply:
sometimes I do `.locals()` to make it obvious it is a method

Reply:
As for `returns an extended locals with living status when a tenant is currently living in the property` and `returns an extended locals which includes the first tenant\'s name when a tenant is NOT living in the property` use `describe`'s to break this up. 

```js
describe('when the tenant lives in property', () => {
  it('living status', () => {
  })
})
```
```js
describe('when the tenant does not live in property', () => {
  it('returns the first tenant name', () => {
  });
})
```
And..

`it('extends from super.locals', () => {});`

Reply:
Good idea, thanks @JoeChapman 👍